tools/dump_kubernetes.sh

```diff
@@ -46,6 +46,10 @@ usage() {
   error '  -m, --max-bytes          max total bytes, 0=no limit, default='${DEFAULT_MAX_LOG_BYTES}
   error '  -l, --label              if set, dump logs only for pods with given labels e.g. "-l app=pilot -l istio=galley"'
   error '  -n, --namespace          if set, dump logs only for pods in the given namespaces e.g. "-n default -n istio-system"'
+  error '  -p, --pod                if set, dump logs only for pod matching these values e.g. "-n istio-system -p istio-policy-7857db5f54-wj59d -p istio-pilot-5cb9d848b5-xx25q"'
+  error '  -c, --control-plane      if set, dump debugging information from control plane'
```

Comment:
```
$ ./tools/dump_kubernetes.sh -c
Error from server (MethodNotAllowed): the server does not allow this method on the requested resource
```

Reply:
This also takes a long time, are we collecting too much info?

Reply:
The lengthy part is to dump the kubectl describe of each api-resources.
This was to effectively implement the non-working command you asked on original issues
`kubectl get describe -n istio-system`

If you prefer we could just set a fixed list of the most important api-resources to describe?